pkg/app/master/tui/tui.go

```diff
@@ -17,6 +17,9 @@ func RunTUI(model tea.Model, standalone bool) {
 		os.Exit(1)
 	}
 	defer f.Close()
+
+	log.SetOutput(f)
```

Comment:
probably don't need to configure logging. should be ok to just start using it. the global parameter handling logic should configure it already, but it's good to double check :)